src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -1556,6 +1555,231 @@ public void testLeadingDeletionInAltHaplotype() {
         Assert.assertTrue(has28BaseDeletion);
     }
 
+    @Test
+    public void testGvcfBeforeRebase() throws Exception {
```

Comment:
Can you add a comment explaining which "rebase" these "BeforeRebase()" test cases are referring to?

Reply:
ilya

Reply:
Fixed